src/index.js

```diff
@@ -73,6 +99,12 @@ module.exports = class {
       throw new Error(`Invalid timezone string ${timezone}.`);
     }
 
+    validatePart(this.minute, validators["minute"]);
+    validatePart(this.hour, validators["hour"]);
+    validatePart(this.dayOfMonth, validators["dayOfMonth"]);
+    validatePart(this.month, validators["month"]);
+    validatePart(this.dayOfWeek, validators["dayOfWeek"]);
```

Comment:
Minor: is the `validators` object needed if we're only using it once here?

E.g. instead of 

```js
validatePart(this.minute, validators["minute"]);
```

it could just be

```js
validatePart(this.minute, { min:0,max:59 });
```

Reply:
Hi. In the new changes im pushing, the use of validators object will be more accurate.